Spec/README.md

```diff
@@ -513,20 +520,15 @@ This file must include:
 Sample rendering:
 ![Description HTML Styles Sample Rendering](DESCRIPTION_HTML_STYLES.png)
 
-**[FONTLOG.txt](https://github.com/googlefonts/OswaldFont/blob/master/FONTLOG.txt):** A log file which lists changes to each release.
+**[promo.png](https://github.com/indestructible-type/Bodoni/tree/master/documentation)**: In order to tweet about a new release, Google Fonts needs 2-3 pictures, different from the ones used in the README.md. If the Family is variable, an animation is welcome.
 
-**[OFL.txt](https://github.com/googlefonts/OswaldFont/blob/master/OFL.txt):** The OFL license file. The first line of the license file must contain the font family's copyright string. Copyright notices should match a pattern similar to: "Copyright 2019 The Familyname Project Authors (git url)". It must not include © copyright sign since the CFF table copyright notice key is ascii only.
-
-**[README.md](https://github.com/TypeNetwork/Roboto):** contains information about the font family and instructions on how to build the family.
-
-**[fonts](https://github.com/googlefonts/OswaldFont/tree/master/fonts):** Directory containing font binaries or subdirectories for each font format. If your project is going to provide multiple formats, do not include them all in one folder. Create a folder for each format e.g `fonts/otf`, `fonts/ttf`.
+**[fonts](https://github.com/googlefonts/OswaldFont/tree/master/fonts):** Directory containing font binaries or subdirectories for each font format. If your project is going to provide multiple formats, do not include them all in one folder. Create a folder for each format e.g `fonts/otf`, `fonts/ttf`, `fonts/woff2`.
```

Comment:
Please please please stop encouraging font projects to keep generated binary font artifacts in Git when they have the sources and build systems there too! You don't use those generated files on Google Fonts anyway and the practice makes for a terrible workflow experience when people try to collaborate via Git.

Reply:
I agree. I've never liked the binaries being included in the repo since they're impossible to diff and I also think a CI/CD should take care of building the fonts. However, atm, I think we only have a couple of projects which use GH actions or travis to build the fonts. In the near future, I would like us to use FontShip since it uses GH actions and our existing scripts/tooling. This won't happen immediately but we will get there. For the time being I don't mind this being merged.

Reply:
I only modified the last sentence of this paragraph, this spec was already there. Anyway this is for the upstream repo, and we are just saying "if" you want other formats, this is how you should organise them.  But I do think the binaries should be available in the upstream repo if the upstream had them at the first place, and I don't think we should expect users to build the fonts themselves: the build script is for the GF and the designer/foundry.